sycl/include/sycl/ext/intel/esimd/memory.hpp

```diff
@@ -1107,6 +1146,10 @@ __ESIMD_API simd<T, m * N> media_block_load(AccessorTy acc, unsigned x,
   static_assert(m <= 64u, "valid block height is in range [1, 64]");
   static_assert(plane <= 3u, "valid plane index is in range [0, 3]");
 
+#ifdef ESIMD_FORCE_STATELESS_MEM_ACCESSX
+  static_assert(
```

Comment:
This will fire even if `media_block_load` is not called. As a variant - guard the entire `media_block_load` with `#ifndef __ESIMD_FORCE_STATELESS_MEM`

Reply:
Right, I forgot to fix this place. I will guard both functions media_block_load() and media_block_store() with "#ifndef __ESIMD_FORCE_STATELESS_MEM". That should produce compilation errors on attempts to use them rather than having runtime errors.